app/views/trends/index.html.erb

```diff
@@ -2,22 +2,9 @@
 <% title_actions display_link_if_authorized(_("Add Trend Counter"), hash_for_new_trend_path),
                  documentation_button('4.1.3Trends') %>
 
-<% if @trends.empty? %>
-  <div class="alert alert-info">
-    <%= alert_close %>
-    <p><strong><%= _("No trend counter defined.") %></strong></p>
-    <p><%= _("To define trend counters, use the Add Trend Counter button.") %><br/>
-      <%= (_("To start collecting trend data, set a cron job to execute 'foreman-rake trends:counter' every Puppet Interval (%s minutes).") % Setting.puppet_interval).html_safe %></p>
```

Comment:
On Sun, Jan 3, 2016 at 10:33 AM, alon goldboim <notifications@github.com>
wrote:

> > - <p><%= _("To define trend counters, use the Add Trend Counter
> button.") %><br/>



^^ this line.